src/aai/aai.js

```diff
@@ -97,6 +114,24 @@
         return djc;
       } //if jQuery(selector)
     };
+
+    function getCookie(cookieName) {
```

Comment:
isn't there a utility function that handles cookies? I have seen handling cookies multiple times

Reply:
Removed. I forgot to remove it.